index.html

```diff
@@ -10,13 +10,22 @@
   <link rel="stylesheet" href="//unpkg.com/docsify/themes/buble.css">
 </head>
 <body>
+    <nav>
+      <a href="#/">Home</a>
+      <a href="#/guides/players/readme.md">Player Guides</a>
+      <a href="#/guides/tactics/readme.md">Tactics</a>
+      <a href="#/guides/roles/readme.md">Roles</a>
+      <a href="#/guides/makers/readme.md">Maker Guides</a>
+    </nav>
   <div id="app"></div>
   <script>
     window.$docsify = {
       name: 'Synixe',
       repo: 'Synixe/Documents',
+      logo: 'https://cdn.discordapp.com/avatars/403101852771680258/3851b8701c42218e485dae9fe32d51e9.webp?size=512',
```

Comment:
Using a URL for images is unacceptable no?

Reply:
This is a link to the actual server icon, which is not at risk of being deleted

Reply:
This is a link to the actual server icon, which is not at risk of being deleted

Reply:
Still could have used a file path though